lib/logitech_receiver/notifications.py

```diff
@@ -136,6 +151,24 @@ def _process_hidpp10_custom_notification(device, status, n):
 
 	_log.warn("%s: unrecognized %s", device, n)
 
+# TODO: find somewhere proper to put this
```

Comment:
@pwr where would be a good place to put some code like this (run at startup, revert on close)?